things/postgres/channels.go

```diff
@@ -342,10 +341,10 @@ func (cr channelRepository) hasThing(chanID, thingID string) error {
 }
 
 type dbChannel struct {
-	ID       string `db:"id"`
-	Owner    string `db:"owner"`
-	Name     string `db:"name"`
-	Metadata string `db:"metadata"`
+	ID       string `dm:"id"`
+	Owner    string `dm:"owner"`
+	Name     string `dm:"name"`
+	Metadata string `dm:"metadata"`
```

Comment:
`db` is not related to the database instance, this is tag that is used in `sqlx` lib. Please these go tags to the old version `db:"id"`, `db:"owner"`, etc. Same goes for the `users`.